### ISSUE TITLE
Shadows on structural components w/ designToken support

### ISSUE THREAD

**Issue by samiresua** [contentful/experience-builder#539]
Would be interested to find out if there are there plans to add support for shadows to structural components, would be nice to see some designToken support for them too if that is the case?

**Comment by elylucas** [comment]
Yes, we are working on adding drop shadow support to our structural/basic components and added as an option for your own custom components. I'll ping this issue here when its available.

Internal ticket for Contentful people to reference: https://contentful.atlassian.net/browse/ALT-745
